crypto/aes/build.info

```diff
@@ -38,6 +38,8 @@ GENERATE[aesv8-armx.S]=asm/aesv8-armx.pl $(PERLASM_SCHEME)
 INCLUDE[aesv8-armx.o]=..
 GENERATE[vpaes-armv8.S]=asm/vpaes-armv8.pl $(PERLASM_SCHEME)
 
+GENERATE[aes-armv4.S]=asm/aes-armv4.pl $(PERLASM_SCHEME)
+GENERATE[bsaes-armv7.S]=asm/bsaes-armv7.pl $(PERLASM_SCHEME)
```

Comment:
Thanks for hints. While debugging this issue, not only `Makefile` but also generated asm files are need to be cleaned. It was very annoying for me that `make clean` does not remove generated asm files. I will submit a new PR to clean them. Please look at it.